revolv/static/main.scss

```diff
@@ -16,4 +16,21 @@ p, h2, h3, h4 {
     font-family: $body-font-family;
 }
 
+// TODO: make this less hacky
```

Comment:
Why do you need to do the !important stuff? Even if it's hacky, I think we should at least put why it needs to be hacky in a comment here